scaper/core.py

```diff
@@ -1698,38 +1698,42 @@ def _generate_audio(self, audio_path, ann, reverb=None,
 
         with _set_temp_logging_level(temp_logging_level):
 
-            # Array for storing all tmp files (one for every event)
-            tmpfiles = []
-            with _close_temp_files(tmpfiles):
-                isolated_events_audio_path = []
+            # List for storing all generated audio (one array for every event)
+            event_audio_arrays = []
+            isolated_events_audio_path = []
+            duration_in_samples = int(self.duration * self.sr)
 
-                role_counter = {'background': 0, 'foreground': 0}
+            role_counter = {'background': 0, 'foreground': 0}
```

Comment:
what's the role of `role_counter`? doesn't seem to be used anywhere else?

Reply:
`role_counter` is used when saving the isolated event audio. Here: https://github.com/justinsalamon/scaper/blob/ea72a7c7a67544b2df42395721f4572fe5bfa85e/scaper/core.py#L1844

Reply:
This showed up in the diff for some mysterious reason.